stencil.config.ts

```diff
@@ -4,13 +4,18 @@ import { sass } from '@stencil/sass';
 import { Config } from '@stencil/core';
 import { basename, dirname, join, resolve } from 'path';
 import { existsSync, readdirSync, readFileSync, unlinkSync, writeFileSync } from 'fs';
+import { reactOutputTarget } from '@stencil/react-output-target';
 import ts from 'typescript';
 
 export const config: Config = {
   globalScript: 'src/global/global.ts',
   globalStyle: 'src/global/global.shared.scss',
   namespace: 'lyne-components',
   outputTargets: [
+    reactOutputTarget({
+      componentCorePackage: 'lyne-components',
+      proxiesFile: '../src/components/components.ts',
```

Comment:
Should that not be added to a new folder? (https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/react-output-target/README.md and https://stenciljs.com/docs/react)

Reply:
Also it seems that there are more steps required (https://stenciljs.com/docs/react) (?)